src/states/north_dakota/main.py

```diff
@@ -0,0 +1,37 @@
+import re
+from bs4 import BeautifulSoup
+from common import cache_request, to_list
+
+BASE_URL = 'https://vip.sos.nd.gov/CountyAuditors.aspx'
+
+re_phfaxemail = re.compile(
+  r'\s*(?P<phone>\(?\s*\d{3}\s*\)?\s*\d{3}-?\d{4})\s*(?:.*?)?\n'  # ignore extension
+  + r'\s*(?P<fax>\(?\s*\d{3}\s*\)?\s*\d{3}-?\d{4})\s*\n'
+  + r'\s*(?P<email>(?:[\w\-\.]+)@(?:[\w\-\.]+)\.(?:[a-zA-Z]{2,5}))\s*',
+  re.MULTILINE
+)
+
+
+def parse_row(row):
+  cols = [col.get_text('\n').strip() for col in row('td')]
+  county = f"{cols[1]} County"
+  phfaxemail = re_phfaxemail.match(cols[3]).groupdict()
+  return {
+    'locale': county,
+    'county': county,
+    'official': cols[2],
+    'phones': to_list(phfaxemail.get('phone')),
```

Comment:
Do we check (either here or later on) whether we are able to get the fax / email?  It looks like we should always have one.

Reply:
The regex will fail to match if either one is missing, throwing an error.

Reply:
LGTM